packages/quantification/src/utils.ts

```diff
@@ -1,5 +1,30 @@
-import { multiply } from '@nori-dot-com/math';
+import { divide, multiply } from '@nori-dot-com/math';
+
+import { ATOMIC_WEIGHT_RATIO_OF_CO2_TO_C } from './constants';
 
 export const convertM2ToAcres = ({ m2 }: { m2: number }): number => {
   return multiply(m2, 0.000_247_105_381_467_17);
 };
+
+/**
+ * Converts grams per meter squared and a map area in meters squared to the total tonnes of CO2e
+ *
+ */
+export const convertGramsPerM2ToTonnes = ({
+  gramsPerM2,
+  mapUnitAreaInM2,
+}: {
+  gramsPerM2: number;
+  mapUnitAreaInM2: number;
+}): number => {
+  const gramsOfCarbonForMapUnitInM2 = multiply(gramsPerM2, mapUnitAreaInM2);
+  const tonnesOfCarbonForMapUnit = divide(
```

Comment:
fwiw the `unit` utility from mathjs is better suited for this purpose. I'm posting this just as a reminder for the future, I don't think that change needs to happen here